src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -5349,4 +5349,26 @@ public static boolean isPasswordValidatorReconfigurationEnabled()
     {
         return conf.password_validator_reconfiguration_enabled;
     }
+
+    public static long getInvalidClientDisconnectPeriod(TimeUnit unit)
+    {
+        return conf.invalid_client_disconnect_period.to(unit);
+    }
+
+    public static void setInvalidClientDisconnectPeriod(TimeUnit unit, long duration)
```

Comment:
are we expecting to update these values from somewhere? I don't see any call sites for this method